appengine/standard/firebase/firetactoe/templates/_firebase_config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
If you change this file into json, then you also need to change the way that the firebase config is included and initialized in the `fire_index.html` template.